### ISSUE TITLE
Intel driver

### ISSUE THREAD

**Issue by Mario7M** [bzminer/bzminer#128]
Trying mine kaspa on bzminer ver12.1.1. Miner start with message Intel driver not found. I have AMD CPU. What is wrong?

**Comment by FreeEagle2020** [comment]
Yes, I got this issue as well mining kaspa but I am using Intel Celeron CPU 847 on BTC-S37 motherboard.  I also get an error message "Watchdog detected an issue. Bzminer restarted XX times".

**Comment by Mario7M** [comment]
I have AMD CPU and nvidia 3070 non LHR GPUs so I don`t understand this message: "Intel driver don`t found" after which miner doesn`t start mining.

**Comment by ttonybony** [comment]
I am having the same problem.
